src/PBI-Tools/PowerBI/AnalysisServicesServer.cs

```diff
@@ -192,15 +192,24 @@ public void Dispose()
 
             _proc = null;
             if (debugMode) return;
+            var failedConfigFound = false;
 
-            if (Log.IsEnabled(LogEventLevel.Debug))
+            foreach (var file in Directory.EnumerateFiles(_tempPath, "*", SearchOption.AllDirectories))
             {
-                foreach (var file in Directory.EnumerateFiles(_tempPath, "*", SearchOption.AllDirectories))
+                if (file.ToLower().Contains("failed")) {
+                    failedConfigFound = true;
+                    Log.Warning($"Config folder contains file {file}. Probably MSMDSRV.exe was unable to launch because of an invalid config. Deletion of the config files will skipped to allow inspection.");
```

Comment:
Serilog doesn't use string interpolation. Instead, it uses semantic logging where parameters are explicitly declared and provided using traditional `params object[]` syntax.
In this case it would be:
```
Log.Warning($"Config folder contains file {FailureFilePath}. Probably MSMDSRV.exe was unable to launch because of an invalid config. Deletion of the config files will skipped to allow inspection.", file);
```